### PR TITLE
Pre-lunch edits.

### PR DESCRIPTION
**Pull request by molpopgen**
Some minor edits made while rereading the paper.  Mostly clarity/shortening.  This helps me get my head back in it.

### CODE REVIEW

File: treestats_paper.tex [hunk 1]
```diff
@@ -353,12 +353,13 @@ \section*{Framework and statistics}
 \subsection*{Node statistics}
 
 Perhaps the most natural way to summarize weights on the tree
-is simply to examine the values at each node.
-This would allow us, for instance, to
-count the number of samples that inherit from each node in the tree.
+is to examine the values at each node.
+\krt{I hope I didn't make this bit incorrect?  I found myself having to reread it several times.}
```
Comment (jeromekelleher) [inline treestats_paper.tex]: I think this isn't quite right @molpopgen --- isn't the point that the simplest f we can have to just return the weights directly (lambda x: x)? The resulting statistic then tells us how many samples inherit from each node in the tree.

Comment (molpopgen) [reply treestats_paper.tex]: Yeah, that's right.  I'll try to tweak this further.  The description is quite abstract, I think, and could perhaps be simpler.

Comment (molpopgen) [reply treestats_paper.tex]: The i.e. are Latin, hence the emph.  But yeah, that'd probably be done by the journal.

Comment (molpopgen) [reply treestats_paper.tex]: I mostly reverted this change & tried to make it a bit more explicit.

**Comment by jeromekelleher** [comment]
Did you mean to close this @molpopgen?

**Comment by molpopgen** [comment]
yes--I figure it is better to work on this when everyone is available.

**Pull request closed without merging**